apps/style/applab/style.scss

```diff
@@ -206,6 +206,7 @@ div#visualizationResizeBar {
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
+  cursor: crosshair;
```

Comment:
Any particular reason you're overriding the `ew-resize` cursor set [here in common.scss](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/style/common.scss#L298]?